pkg/skaffold/cluster/minikube.go

```diff
@@ -93,9 +103,46 @@ func minikubeExec(arg ...string) (*exec.Cmd, error) {
 	if err != nil {
 		return nil, fmt.Errorf("getting minikube executable: %w", err)
 	}
+
+	minVer, err := supportsUserFlagFunc()
```

Comment:
It would make sense for `minikubeBinaryFunc` to return the binary path as well as version so you don't have to return err twice. 
1) minikube binary issue
2) minikube version parsing.